packages/vitest/src/node/reporters/verbose.ts

```diff
@@ -9,45 +10,62 @@ export class VerboseReporter extends DefaultReporter {
   protected verbose = true
   renderSucceed = true
 
-  printTask(task: Task): void {
+  printTestModule(module: TestModule): void {
+    // still print the test module in watch mode,
```

Comment:
This is not watch-mode specific, it's also done on run-mode in TTY:

```suggestion
    // still print the test module in TTY,
```